File: src/shared/py/langboard_shared/domain/models/BotTriggerGraphApprovalRequest.py
from .bases import BaseGraphApprovalBotRequest
from .GraphApprovalRequest import GraphApprovalOriginType


class BotTriggerGraphApprovalRequest(BaseGraphApprovalBotRequest, table=True):
    @staticmethod
    def get_request_type() -> GraphApprovalOriginType:
        return GraphApprovalOriginType.Trigger

File: src/socket/src/models/BotTriggerGraphApprovalRequest.ts
import BaseGraphApprovalBotRequest from "@/models/bases/BaseGraphApprovalBotRequest";
import { Entity } from "typeorm";

@Entity({ name: "bot_trigger_graph_approval_request" })
class BotTriggerGraphApprovalRequest extends BaseGraphApprovalBotRequest {}

export default BotTriggerGraphApprovalRequest;
